pull-requests.md

```diff
@@ -0,0 +1,164 @@
+  These issues are usually found by users of MDN Web Docs.
+- **Content bug fixes:** usually done by volunteers to close issues on `mdn/content` repo.
+
+Regardless of how content changes are done, they are submitted as pull requests on GitHub. The content changes go through the following stages before they are published on MDN Web Docs:
+
+1. **Submission:** As a pull request author, you submit the changes. There are a few guidelines you need to be aware of before and after you open a pull request.
+2. **Review:** Your changes are reviewed by MDN staff members and volunteers.
+   When a pull request related to a particular topic is opened, it will be assigned to that area's topic review owners.
+   The `CODEOWNERS` file specifies the teams who own certain content directories.
+3. **Approval:** The changes can be merged after a approving review has been made.
+   The assigned reviewer will merge the pull request.
+4. **Publishing:** Updated content goes live within a day of merging via a site rebuild once every 24 hours.
+
+### Decision tree
+
+TODO: example decision tree in mermaid
+
+```mermaid
+graph TB
+  A("Do you think<br> online learning is right for you?")
+  B("Do you have time to design<br> a service learning component?")
+  C("What is the civic or public purpose of your discipline?<br> How do you teach that without service learning?")
+  D("Do you have departmental or school<br> support to plan and implement service learning?")
+  E["Are you willing to be a trailblazer?"]
+  F["What type of service learning to you want to plan?"]
+
+  A==Yes==>B
+  A--No-->C
+  B==Yes==>D
+  B--No-->C
+  D--Yes-->F
+  D--No-->E
+  E--Yes-->F
+  E--No-->C
+```
+
+## Guidelines for opening a pull request
+
+### Before you open a pull request
+
+Anyone who plans to open a PR to make a change on MDN Web Docs should:
+
+- **Open a discussion** to describe why you think the changes are required to improve the content.
+  This is important if your PR is a technical change, not typo fixes, grammatical improvements, formatting or structural changes.
+- **File an issue describing what the PR will change.**
+  When you submit a PR, reference this issue using `#` followed by the issue's ID, for example `#1234`.
+
+### Opening a pull request
+
+- **PRs should be short and focused to one issue** or a related set of changes.
+  If a PR becomes too large, the reviewer may close your PR and ask that you submit a separate PR for each logical set of changes that belong together.
+- **PRs should not contain large amounts of grammar updates**.
+  MDN Web Docs contains technical documentation; you should not report improvements in the grammar, but cases where the grammar is incorrect are acceptable.
+- **PRs that update API usage need an accompanying mdn/content PR** to update the corresponding documentation.
+  PRs of this type may be rejected if there is no corresponding content PR.
```

Comment:
I think this is suppose to be concerning if demo (interactive examples, dom examples, learn area et al) updates are needed.

If demo updates happen is there a corresponding content pr
If content and in page code examples are updated, are the corresponding demos updated